src/utils/protocol.ts

```diff
@@ -11,7 +11,7 @@ function _calculatePercentageDifference(vOneBigInt: BigInt, vTwoBigInt: BigInt):
   const ret = numerator.div(denominator).times(BigDecimal.fromString("100"));
 
   // there is no absolute value method on BigDecimal so its simulated here
-  if (ret.lt(BigDecimal.zero())) {
+  if (ret.lt(BigDecimal.zero()) == true) {
     return ret.times(BigDecimal.fromString("-1"));
   } else {
     return ret;
```

Comment:
Can we remove the else?  And should we do === ?

Reply:
as there is no absolute value method, if the percentage difference is below zero we want to multiply by negative one to simulate it - this could be changed to:
```
let ret;
if (ret.lt(BigDecimal.zero()) == true) ret = ret.times(BigDecimal.fromString("-1"));
return ret;
```
If you think this method is better than I can change it.

Reply:
This is AssemblyScript which means it unfortunately has some caveats compared to Typescript - there are some [old threads](https://github.com/AssemblyScript/assemblyscript/issues/621#issuecomment-497973428) on `==` vs `===` and how they differ, but not much that I've found in the docs recently. If the language still behaves the same way as in the threads, then `ret.lt(BigDecimal.zero()) === ret.lt(BigDecimal.zero())` would be `true` whereas `ret.lt(BigDecimal.zero()) === true` would be `false`

Reply:
It's still safe enough to try however, lmk if you believe it's worth giving a go @bitcoinbrisbane